tests/parm/nems.configure.hafs.IN

```diff
@@ -0,0 +1,17 @@
+ EARTH_component_list: ATM
+
+ ATM_model:            fv3
+ ATM_attributes::
+ :: 
+
+ runSeq::
+   ATM
+ ::
+
+ DRIVER_attributes::
+   start_type = @[RUNTYPE]
+ ::
+
+ ALLCOMP_attributes::
+   mediator_read_restart = @[USE_COLDSTART]
```

Comment:
This looks to me a  standalone fv3 run, why do we need the "mediator_read_restart"?

Reply:
@junwang-noaa I am not sure but without those the standalone FV3 fails. It was working before without those extra namelist options. We did not do anything special for it. It might be inherited from some other changes in UFS level. So, I just need to adjust it to make HAFS specific RTs work.

Reply:
@junwang-noaa and @uturuncoglu, I think these are needed only when we try to build one HAFS application (-DAPP=HAFS) forecast executable to be able to run various different atm-only, atm-ocn, atm-wav, and HAFS-cdeps coupling configurations. If you build with -DAPP=ATM, then most likely you do not need these extra items.